src/commands/reload.ts

```diff
@@ -125,15 +130,36 @@ export default class Reload extends Command {
     }
 
     /**
-     * Reloads a non-Node.js service by rebuilding and restarting it.
+     * Reloads a Nginx service by restarting container.
      * @param serviceName - Name of the service to reload
      */
-    private async reloadNonNodeService (serviceName: string): Promise<void> {
+    private async reloadNginxService (serviceName: string): Promise<void> {
+        this.log(`Service: ${serviceName} restarted`);
+        await this.dockerCompose.restart(serviceName);
+    }
+
+    /**
+     * Reloads a Java service by rebuilding and restarting it.
+     * @param serviceName - Name of the service to reload
+     *  @param flags - Flags passed to the command
+     */
+    private async reloadJavaService (serviceName: string, flags: any): Promise<void> {
+        const codeHashFile = this.getCodeHashFile(serviceName);
+        if (flags.force && codeHashFile) {
+            unlinkSync(codeHashFile);
+        }
         this.dependencyCache.update();
```

Comment:
As a point of note, I think this can be removed unless people are still using v1 of Java builder - which I don't think is possible/the case (separate PR though)

Reply:
Its still useful. Even if there are no changes and you want to force a reload and also useful for troubleshooting.